Description: allow increase in c value of ATK_STATE_LAST_DEFINED
 The new version of atk increased the value of ATK_STATE_LAST_DEFINED,
 upstream did not bump the soversion, so clearly they do not consider
 this a change that is likely to break downstream applictions.

 Futhermore the only rust code I can find that actually uses the
 constant in question is some code in rust-atk that translates it
 to an enum I cannot find any rust code that makes use of the 
 corresponding enum value either.

Author: Peter Michael Green <plugwash@debian.org>

--- rust-atk-sys-0.14.0.orig/tests/abi.rs
+++ rust-atk-sys-0.14.0/tests/abi.rs
@@ -134,6 +134,14 @@ fn cross_validate_constants_with_c() {
             continue;
         }
 
+        if *rust_name == "(gint) ATK_STATE_LAST_DEFINED" {
+            // ATK_STATE_LAST_DEFINED may increase in newer versions of the atk library.
+            if c_value.parse::<i32>().unwrap_or(0) > rust_value.parse::<i32>().unwrap_or(i32::MAX) {
+                results.record_passed();
+                continue;
+            }
+        }
+
         if rust_value != c_value {
             results.record_failed();
             eprintln!(
